debian/control

```diff
@@ -27,6 +27,10 @@ Depends:
  ${misc:Depends},
  python3-opencv,
  python3-pitop (= ${binary:Version}),
+ libopenblas-dev,
+# Required for face detection & emotion mapping
```

Comment:
Which is not a dependency of the core SDK. It's part of `-full`

Reply:
not sure what you mean @m-roberts

Reply:
Face detection and emotion mapping are not core dependencies of the SDK. They are dependencies of OpenCV, so should move to `-full`.